aspnetcore/client-side/blazor/index.md

```diff
@@ -51,7 +51,7 @@ DESCRIPTION
 
 ## Suggest a feature or file a bug report
 
-To make suggestions and file bug reports, open an issue in the [aspnet/Blazor repo](https://github.com/aspnet/Blazor/issues/new). For general help and to ask questions, hit up the devs on the [Blazor Gitter chat](https://gitter.im/aspnet/Blazor).
+To make suggestions and file bug reports, open an issue in the [aspnet/Blazor repo](https://github.com/aspnet/Blazor/issues/new). For general help and to ask questions, hit up the devs on the [Gitter Blazor room](https://gitter.im/aspnet/Blazor) or [Slack Blazor channel](https://aspnetcore.slack.com/messages/C958WJP2N/) ([signup](http://tattoocoder.com/aspnet-slack-sign-up/)).
```

Comment:
The product team hangs out on Gitter, but not on Slack, so I don't think we should promote the Slack channel here. How about:

To make suggestions and file bug reports, please [open an issue](https://github.com/aspnet/blazor/issues). For general help and to ask questions from the community join the conversation on [Gitter](https://gitter.im/aspnet/blazor).